### PR TITLE
fix: title and translation

### PR DESCRIPTION
**Pull request by ricardotenorio**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/portuguese/java/loops/index.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
 ---
 title: Loops
-localeTitle: rotações
+localeTitle: Loops
 ---
-# rotações
+# Loops
 
-Sempre que você precisar executar um bloco de código várias vezes, um loop geralmente vem a calhar.
+Sempre que você precisar executar um bloco de código várias vezes, um loop(ou laço) geralmente vem a calhar.
```
Comment (CleoAguiar) [inline guide/portuguese/java/loops/index.md]: I suggest to remove the parentheses or the word "ou".

```suggestion
Sempre que você precisar executar um bloco de código várias vezes, um loop(ou laço) geralmente vem a calhar.
```

**Review by CleoAguiar — CHANGES_REQUESTED** [review]
I suggest to remove the parentheses or the word "ou".

File: guide/portuguese/java/loops/index.md (hunk 1 above)
Comment (RandellDawson) [reply guide/portuguese/java/loops/index.md]: I made the change on the user's behalf.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**